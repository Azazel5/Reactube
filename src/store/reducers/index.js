import apiReducer from './api'
import videosReducer from './video'
import {combineReducers} from 'redux'

/** 
 * Redux-thunk is pretty nice if you have simple API calls in you application. However, considering 
 * shifting to redux-saga whenever needed, as it seems to solve those problems. 
*/

export default combineReducers({
    api: apiReducer, 
    videos: videosReducer
})
