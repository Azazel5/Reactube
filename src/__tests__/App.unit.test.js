import React from 'react'
import {shallow} from 'enzyme'
import App from '../App'
import { Provider } from 'react-redux'
import { configureStore } from '../store/configureStore'

const store = configureStore()

describe('<App />', () => {
    test('renders <App /> component', () => {
        const wrapper = shallow(<Provider store={store}><App /></Provider>)
        expect(wrapper).toMatchSnapshot() 
    })
})