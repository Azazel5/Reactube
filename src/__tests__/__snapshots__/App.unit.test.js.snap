// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders <App /> component 1`] = `
<appLayout>
  <Switch>
    <Route
      component={[Function]}
      path="/watch"
    />
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</appLayout>
`;
